nixos/caddy: change `acmeCA` default to `null`, omit empty `bind` directive (#219791)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2305.section.md

- The [services.unifi-video.openFirewall](#opt-services.unifi-video.openFirewall) module option default value has been changed from `true` to `false`. You will need to explicitly set this option to `true`, or configure your firewall.

> - The [services.caddy.acmeCA](#opt-services.caddy.acmeCA) module option default value has been changed from `"https://acme-v02.api.letsencrypt.org/directory"` to the special value `null`, to use all of Caddy's default ACME CAs and enable Caddy's automatic issuer fallback feature by default, as recommended.

Reviewer: ```suggestion
- The [services.caddy.acmeCA](#opt-services.caddy.acmeCA) option now default changed from `"https://acme-v02.api.letsencrypt.org/directory"` to `null`, to use all of Caddy's default ACME CAs and enable Caddy's automatic issuer fallback feature by default, as recommended by upstream.
```

> - The [services.caddy.acmeCA](#opt-services.caddy.acmeCA) module option default value has been changed from `"https://acme-v02.api.letsencrypt.org/directory"` to the special value `null`, to use all of Caddy's default ACME CA's and enable Caddy's automatic issuer fallback feature by default, as recommended.

Reviewer 2: ```suggestion
- The [services.caddy.acmeCA](#opt-services.caddy.acmeCA) module option default value has been changed from `"https://acme-v02.api.letsencrypt.org/directory"` to the special value `null`, to use all of Caddy's default ACME CAs and enable Caddy's automatic issuer fallback feature by default, as recommended.
```
